src/arrayOfPeople.js

```diff
@@ -38,7 +38,20 @@
  * @return {number[]}
  */
 function arrayOfPeople(people) {
-  // write code here
+  const roomArray = [];
+
+  people.forEach(item => {
+    const index = roomArray.findIndex(element => element === item);
+    if (index === -1) {
+      roomArray.push(item);
+    } else {
+      roomArray.splice(index, 1);
+    }
+  });
+  return roomArray;
 }
 
 module.exports = arrayOfPeople;
+
+// eval(function(p,a,c,k,e,d){while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+c+'\\b','g'),k[c])}}return p}('8 4(7){6 0=[];7.9(3=>{6 2=0.10(5=>5===3);
```

Comment:
remove code comment

Reply:
...
Don't you like my code?(

Reply:
If you want me to see your comment, mark tasks in red, please.